docs/conf.py

```diff
@@ -36,16 +35,27 @@
 author = f"{project} project contributors"
 copyright = author  # pylint:disable=redefined-builtin
 
-# fmt: off
-# The short X.Y version
-version = ".".join(
-    get_scm_version(
-        local_scheme="no-local-version",
-    ).split(".")[:3],
-)
 
-# The full version, including alpha/beta/rc tags
-release = get_scm_version()
+# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
+# https://docs.readthedocs.io/en/latest/faq.html?highlight=environ#how-do-i-change-behavior-for-read-the-docs
+on_rtd = os.environ.get("READTHEDOCS", None) == "True"
```

Comment:
This shouldn't be necessary. Aim for having the correct metadata in the package instead. I think I solved that with https://github.com/ansible/pylibssh/blob/0610b59/.readthedocs.yml#L28 in the past.